force-app/test/jest-mocks/lightning/uiRecordApi.js

```diff
@@ -0,0 +1,35 @@
+/*
```

Comment:
Is there a particular reason why you're not using the official stub for this?

Reply:
implemented using the official stub